src/Microsoft.Identity.Web/Resource/AadIssuerValidator.cs

```diff
@@ -192,16 +245,5 @@ private static string GetTenantIdFromIss(string iss)
 
             return string.Empty;
         }
-
-        /// <summary>
-        /// This method is now Obsolete.
-        /// </summary>
-        /// <param name="aadAuthority">Aad authority.</param>
-        /// <returns>NotImplementedException.</returns>
-        [Obsolete(IDWebErrorMessage.AadIssuerValidatorGetIssuerValidatorIsObsolete, true)]
```

Comment:
I guess it's ok to remove this obsolete method.

Reply:
can add it back if you want.